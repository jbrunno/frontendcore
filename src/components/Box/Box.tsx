import React from 'react';
import { Box as MuiBox, BoxProps as MuiBoxProps } from '@mui/material';

export type BoxProps = MuiBoxProps;

export function Box(props: BoxProps) {
  return <MuiBox {...props} />;
}

export default Box;
