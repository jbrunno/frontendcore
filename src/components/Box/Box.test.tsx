import React from 'react';
import { render } from '@test-utils/component-testing';
import { Box } from './Box';

test('renders the Box', () => {
  const { container } = render(<Box />);
  expect(container).toMatchSnapshot();
});
