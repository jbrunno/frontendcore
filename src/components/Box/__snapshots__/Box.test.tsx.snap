// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the Box 1`] = `
<div>
  <div
    class="MuiBox-root emotion-0"
  />
</div>
`;
